templates/resources/metal_connection.md.tmpl

```diff
@@ -30,6 +30,14 @@ Use this resource to request the creation an Interconnection asset to connect wi
 
 {{tffile "examples/resources/metal_connection/example_3.tf"}}
 
+### Shared Connection with authorization_code Non-redundant NIMF connection from Equinix Metal to a Cloud Service Provider via Equinix Fabric Port
+
+{{tffile "examples/resources/metal_connection/example_4.tf"}}
```

Comment:
Nit: the docs migrator doesn't know the purpose of individual example files, but we do; I recommend using more descriptive file names for these example files, such as `shared_nimf_to_csp.tf` and `shared_nimf_from_fcr.tf` instead of `example_<n>.tf`.  The only time we should use a generic name for an example file is when there is a single example, called `example.tf`, since that one will be automatically discovered by the generator (with the caveat that the generator has a weird inconsistency in the conventional paths for examples, mentioned in #724).

Reply:
I have renamed the metal and fabric connection example.